### PR TITLE
#1157 - cards grid

### LINKED ISSUE #1157: Cards grid
https://github.com/comicrelief/pattern-lab/issues/251

### PR DESCRIPTION
**Pull request by gusliedke**
Fixes #1157 

- change pattern-lab version number once the pattern-lab PR has gone in (https://github.com/comicrelief/pattern-lab/pull/258).

### CODE REVIEW

**Comment by gusliedke** [comment]
@pradip3271 this is the PR and ticket in the description

**Comment by gusliedke** [comment]
@pradip3271 can you review again please https://pr-1159-q375ivy-tx3mbsqmxtu74.eu.platform.sh/fundraise

**Comment by pvhee** [comment]
@gusliedke what is the status of this PR? Maybe @Saphyel can help out with the Travis issues?

**Comment by gusliedke** [comment]
There is one issue in ipad to be investigated. It's on hold until Alex is back to merge it.

**Comment by Heleen-cr** [comment]
passed QA

File: modules/cr_cards/config/install/field.field.paragraph.cr_cards.field_cr_cards_background.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ id: paragraph.cr_cards.field_cr_cards_background
 field_name: field_cr_cards_background
 entity_type: paragraph
 bundle: cr_cards
-label: 'Cards row Background colour'
```
Comment (AndyEPhipps) [inline modules/cr_cards/config/install/field.field.paragraph.cr_cards.field_cr_cards_background.yml]: Are we renaming terminology here?

Comment (Heleen-cr) [reply modules/cr_cards/config/install/field.field.paragraph.cr_cards.field_cr_cards_background.yml]: We're not very consistent in using the term row for every paragraph. In this case it makes sense to not call it a row as it's a grid and can spill over to 2 lines if you 5 cards for example.

Comment (AndyEPhipps) [reply modules/cr_cards/config/install/field.field.paragraph.cr_cards.field_cr_cards_background.yml]: Ok cool, makes sense; just need to make sure Alex et al knows this

**Pull request merged**